controllers/custodian/config.go

```diff
@@ -25,11 +25,12 @@ import (
 const (
 	workersFlagName                     = "custodian-workers"
 	syncPeriodFlagName                  = "custodian-sync-period"
+	requeueIntervalFlagName             = "custodian-requeue-interval"
```

Comment:
This is leaking implementation details to the API. Custodian is the name of an internal controller which could change in future and therefore having that as part of an API should be avoided. We have made that mistake in the past and we should not repeat that now.

Reply:
Sure. Do you think `etcd-status-update-interval` is a better flag name? It talks about exactly what it does, without any controller implementation details. In that case, we can go ahead and introduce it, and phase out `custodian-sync-period`. WDYT?

Reply:
yes we can do that but in v1beta1 we will be renaming `Etcd` to `EtcdCluster` so the status update will be for the `EtcdCluster` and not for `Etcd`. I would leave it to you, if you are willing to change this 2 times then please go ahead. Today we have other flags as well like `custodian-workers` as well. We can change all of that in one go when doing the v1beta1 changes to the API. Till then my recommendation would be keep changes in this PR small and only make changes that are essential.